services/gitlab/gitlab-issues.service.js

```diff
@@ -19,192 +20,44 @@ const queryParamSchema = Joi.object({
   gitlab_url: optionalUrl,
 }).required()
 
-const labelDocumentation = `
-<p>
-  If you want to use multiple labels then please use commas (<code>,</code>) to separate them, e.g. <code>foo,bar</code>.
-</p>
-`
-
 export default class GitlabIssues extends GitLabBase {
   static category = 'issue-tracking'
 
   static route = {
     base: 'gitlab/issues',
-    pattern: ':variant(all|open|closed):raw(-raw)?/:project+',
+    pattern: ':variant(all|all-raw|open|open-raw|closed|closed-raw)/:project+',
```

Comment:
On this badge and the merge requests one, I've refactored this to be a more explicit enum. This does introduce a bit of extra duplication here, but it makes it easier to document in the frontend. It is a slight variant on https://github.com/badges/shields/pull/8967

This does have some knock-on effects to the handle/render methods.

There may be a few other places where we need to do something similar, like the GitHub issue badges.